js/signin.js

```diff
@@ -0,0 +1,104 @@
+const emailInput = document.querySelector("#email-input");
+const passwordInput = document.querySelector("#password-input");
+const emailErrorMessage = document.querySelector("#email-error-message");
+const passwordErrorMessage = document.querySelector("#password-error-message");
+const loginForm = document.querySelector("#login-form");
+
+const eyeOffButton = document.querySelector(".eye-off");
+const eyeOnButton = document.querySelector(".eye-on");
+
+const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6}$/;
+
+// 이메일 입력
+emailInput.addEventListener("focusout", checkEmailInput);
+emailInput.addEventListener("focus", resetInputStyle);
+
+function checkEmailInput() {
+  if (emailInput.value.trim() === "") {
+    showEmailErrorMessage("이메일을 입력해 주세요.");
+  } else if (!validateEmail(emailInput.value.trim())) {
+    showEmailErrorMessage("올바른 이메일 주소가 아닙니다.");
+  } else {
+    hideErrorMessage();
+  }
+}
+
+function validateEmail(email) {
+  return EMAIL_REGEX.test(email);
+}
+
+function showEmailErrorMessage(message) {
+  emailErrorMessage.textContent = message;
+  emailErrorMessage.style.display = "block";
```

Comment:
일관되게 이 부분도 이런 느낌으로 해보면 어떨까 싶어요!
```suggestion
  emailInput.classList.add("show");
```